packages/asl-internal-ui/package.json

```
@@ -10,6 +10,7 @@
     "watch:server": "nodemon -r dotenv/config -e js,jsx,json -w",
     "watch:css": "npm run build:css -- --watch",
     "test": "npm run test:lint && npm run test:deduped",
+    "test:audit": "audit-ci --config .auditrc.json",
```

Unless these args are breaking something we should stick to the setup used prior to merging into the mono repo.

```suggestion
    "test:audit": "audit-ci --high --skip-dev --config .auditrc.json",
```